llvm/test/Assembler/repo-prunning.ll

```diff
@@ -44,16 +39,15 @@ entry:
   ret i32 %1
 }
 
-;CHECK: define available_externally i32 @bar()
 define i32 @bar() {
 entry:
   %call = call i32 @foo()
   ret i32 %call
 }
 
+;CHECK: !TicketNode(name: "me", digest: [16 x i8] c"{{.*}}", linkage: linkonce_odr, visibility: default, pruned: false)
```

Comment:
Is there a reason for the change of order here?

Reply:
The order keeps the same after the RepoMetadataGenerationPass and RepoPruningPass passes. The change is caused by the `Dead Global Elimination` pass which is added after the RepoPruingPass.